style/package.json

```diff
@@ -10,16 +10,25 @@
   "scripts": {
     "config": "cp config.default.js config.js",
     "code_format": "npx prettier --write .",
-    "clean": "rm -rf sprites",
-    "presprites": "npm run clean",
-    "sprites": "mkdir -p sprites && npx spritezero sprites/sprite@2x icons/ --retina && npx spritezero sprites/sprite icons/",
-    "start": "npm run sprites && npx browser-sync -w --port 1776"
+    "clean": "rm -rf dist .parcel-cache",
+    "presprites": "rm -rf dist/sprites",
+    "sprites": "mkdir -p dist/sprites && npx spritezero dist/sprites/sprite@2x icons/ --retina && npx spritezero dist/sprites/sprite icons/",
+    "start": "npm run clean && npm run sprites && npx parcel index.html --open --port 1776",
+    "build": "npm run clean && npm run sprites && npx parcel build index.html --public-url ./"
   },
   "dependencies": {
     "@beyondtracks/spritezero-cli": "^2.3.0"
   },
   "devDependencies": {
-    "browser-sync": "^2.27.7",
+    "maplibre-gl": "https://github.com/ZeLonewolf/maplibre-gl-js/releases/download/v2.0.5/maplibre-gl.tar.gz",
+    "openmapsamples-maplibre": "github:adamfranco/OpenMapSamples-MapLibre",
+    "parcel": "^2.3.0",
     "prettier": "2.3.2"
+  },
+  "engines": {
+    "npm": ">=8.3.0"
```

Comment:
Can you add an .nvmrc file containing something like `stable` for those who use `nvm` to manage their Node versions? Looks like `engines` doesn’t enforce the version that `nvm` uses, unfortunately: nvm-sh/nvm#651.

Reply:
We could do that, but it seems that `stable` is a deprecated alias: https://stackoverflow.com/questions/68357267/nvm-use-stable-sets-node-version-to-older-than-version-installed-by-nvm-insta

We also probably don't want to hard-code to a single Node version, just provide a minimum version, but I'm open to suggestions. :-)

Reply:
8.3 here is also the [NPM version](https://github.com/npm/cli/releases) not the [Node version](https://nodejs.org/en/download/releases/).   I don't think NVM helps with manageing NPM versions, but I could be wrong. NPM can update itself to the latest version with `npm update -g npm`.